.github/workflows/release-drafter.yml

```diff
@@ -20,6 +20,8 @@ on:
   push:
     branches:
       - master
+workflow_dispatch:
```

Comment:
I'd think this needs to be on the same level as `push`.

Reply:
Good spot, thank you.